components/system-broker/pkg/http/correlation_id_transport.go

```diff
@@ -1,44 +0,0 @@
-/*
```

Comment:
Why does this go down the trash

Reply:
There is the same implementation in director that can be used